### PR TITLE
fix(prometheus): clear registry after shutdown

### PR DESCRIPTION
**Pull request by preethivenkatesh-sdase**
Tested in partnerODS -> https://github.com/SDA-SE/partner-ods/pull/1024

### CODE REVIEW

File: sda-commons-server-prometheus/src/main/java/org/sdase/commons/server/prometheus/PrometheusBundle.java [hunk 1]
```diff
@@ -95,8 +95,7 @@ private void createPrometheusRegistry(Environment environment) {
             onShutdown(
                 () -> {
                   Metrics.removeRegistry(meterRegistry);
-                  Metrics.globalRegistry.close();
-                  Metrics.globalRegistry.clear();
+                  CollectorRegistry.defaultRegistry.clear();
```
Comment (sliesche) [inline sda-commons-server-prometheus/src/main/java/org/sdase/commons/server/prometheus/PrometheusBundle.java:98]: `Metrics.globalRegistry.close();`
`Metrics.globalRegistry.clear();`

These codelines are handling micrometer registry, i would prefer to keep them

Comment (preethivenkatesh-sdase) [reply sda-commons-server-prometheus/src/main/java/org/sdase/commons/server/prometheus/PrometheusBundle.java:98]: Oh ok! I will add them back

Comment (preethivenkatesh-sdase) [reply sda-commons-server-prometheus/src/main/java/org/sdase/commons/server/prometheus/PrometheusBundle.java:100]: Micrometer test only complained about not having `Metrics.removeRegistry(meterRegistry);`, so I removed the other code

Comment (preethivenkatesh-sdase) [reply sda-commons-server-prometheus/src/main/java/org/sdase/commons/server/prometheus/PrometheusBundle.java]: updated :)

**Review by sliesche — APPROVED** [review]

**Pull request merged**